girder/molecules/server/nist.py

```diff
@@ -0,0 +1,42 @@
+import re
+import requests
+from bs4 import BeautifulSoup
```

Comment:
We will need to add these to the [requirements.txt](https://github.com/OpenChemistry/mongochemserver/blob/master/requirements.txt)

Reply:
Done https://github.com/OpenChemistry/mongochemserver/pull/96/commits/ccc9310792ab957fd414bf3615d4991ee2f34f9a.